deployments/k8s/helm/signalfx-agent/values.yaml

```diff
@@ -131,7 +131,6 @@ gatherDockerMetrics: true
 # cAdvisor metrics that cause a large increase in DPM for little value for most
 # customers.
 metricsToExclude:
```

Comment:
Similar to above.

Reply:
Since I took the whitelist.json reference out of the metricsToExclude default value in `values.yaml`, it shouldn't use it at all unless people explicitly set `metricsToExclude` to something non-default.